reactjs-films-homework/src/store/rootStore/actorPageStore/getActorData/getActorInfo.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+export interface IGetActorInfo {
+  id: string;
+  lang: string;
+}
+
+const getActorInfo = createAsyncThunk(
+  'getActorInfo/setActorInfo',
+  async (props: IGetActorInfo) => {
+    try {
+      const { lang, id } = props;
+      const baseUrl = `https://api.themoviedb.org/3/person/${id}?api_key=8fa5bc53bb4a09dfb6560253edf33030&language=${lang.toLowerCase()}-${lang}`;
```

Comment:
It's a good practice to move https://api.themoviedb.org and api_key=8fa5bc53bb4a09dfb6560253edf33030&language to common constants so that if something changes you will change the URL or API key only in one place, and it will automatically apply to all the fetch functions.